airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -804,6 +804,13 @@ def generate_scd_type_2_model(self, from_table: str, column_names: Dict[str, Tup
             cdc_cols += f", {cast_begin}{col_cdc_log_pos}{cast_as}" + "{{ dbt_utils.type_string() }}" + f"{cast_end}"
             quoted_cdc_cols += f", {quoted_col_cdc_log_pos}"
 
+        if "_ab_cdc_lsn" in column_names.keys():
```

Comment:
this is copied from the if-statement right above, which enables sorting on `_ab_cdc_log_pos`